### PR TITLE
update context document to remove publicKeyJwk and add JsonWebKey2020

### LINKED ISSUE #612: JwsVerificationKey2020 not in the contexts
The contexts listed in https://identity.foundation/sidetree/spec/#context do not contain `JwsVerificationKey2020`

### PR DESCRIPTION
**Pull request by kdenhartog**
Signed-off-by: Kyle Den Hartog <kyle.denhartog@mattr.global>

This should close #612 and #615

@OR13 can you double check to make sure I did this right?

### CODE REVIEW

File: context-v1.jsonld [hunk 1]
```diff
@@ -16,6 +12,11 @@
       "@id": "sidetree:operation",
       "@type": "@id",
       "@container": "@set"
+    },
+    "JsonWebKey2020": {
```
Comment (OR13) [inline context-v1.jsonld:16]: it should look like this: https://github.com/w3c/did-core-registries/blob/master/contexts/did-v0.11.jsonld#L14

**Comment by kdenhartog** [comment]
This isn't updated anywhere else that I can reference it though. Are you thinking I use something like this?
```
{
 "jwk":"https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v0.0.jsonld",
 "JsonWebKey2020":"jwk:JsonWebKey2020",
 ...
}
```

**Comment by OR13** [comment]
@kdenhartog thats not how JSON-LD works...

A term is defined in a context:

https://github.com/w3c-ccg/security-vocab/blob/master/contexts/security-v2.jsonld#L13

A Vocabulary for the term definition is provided:

https://github.com/w3c-ccg/security-vocab/blob/master/contexts/security-v1.jsonld#L7

The term is defined at the URL:

https://w3c-ccg.github.io/security-vocab/#Ed25519VerificationKey2018

When you can't use `sec`, you must define your own vocabulary, as you did for BBS+:

https://github.com/mattrglobal/jsonld-signatures-bbs-spec/blob/master/contexts/v0.1.jsonld#L6
https://github.com/mattrglobal/jsonld-signatures-bbs-spec/blob/master/contexts/v0.1.jsonld#L10

Here is the current term definition, which must be preserved with a deprecation warning:

https://w3c-ccg.github.io/lds-jws2020/contexts/#JwsVerificationKey2020

The new one would be: https://w3c-ccg.github.io/lds-jws2020/contexts/#JsonWebKey2020

Its definition in here: https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v0.0.jsonld

would look like this:

```
"JsonWebKey2020": "ldssk:JsonWebKey2020",
```

**Comment by OR13** [comment]
ahh, I see you are also trying to implement the nesting feature here... perhapse it would be better to do that here first: https://github.com/w3c-ccg/lds-jws2020

show that tests still pass, and then change the structure of the context definitions.

**Comment by kdenhartog** [comment]
I haven't gotten back to cleaning up this spec yet. Will get around to it within the week (aiming for before our meeting this week).

**Comment by kdenhartog** [comment]
@OR13 I updated this finally, but was a bit confused by one of your comments. This may still be unaddressed.

> "the current term definition, which must be preserved with a deprecation warning" 

To me this implies I should be adding `JsonVerificationKey2020`. I thought we were redefining it so that we didn't have deprecate at this layer and could instead only include `JsonWebKey2020`. Did I misunderstand that?

File: contexts/sidetree-v0.0.0.jsonld [hunk 2]
```diff
@@ -18,6 +15,7 @@
       "@type": "@id",
       "@container": "@set"
     },
-    "usage": "sidetree:usage"
+    "usage": "sidetree:usage",
+    "JsonWebKey2020": "ldssk:JsonWebKey2020"
```
Comment (OR13) [inline contexts/sidetree-v0.0.0.jsonld:19]: Can you add `"@type": "@json"z`

Comment (OR13) [reply contexts/sidetree-v0.0.0.jsonld:19]: if we don't do this, you will errors about `kty` and `kid` not being defined

**Comment by kdenhartog** [comment]
@OR13 fixed

**Pull request merged**